### ISSUE TITLE
[SUGGESTION] Alternate actors for storage of components

### ISSUE THREAD

**Issue by Ikabodo** [misterpotts/fabricate#89]
## Description
Most adventurers would not carry large or heavy resources with them in the field. Many groups solve this by storing items in different kinds of loot sheets or actors. (Like item piles or lootsheet). It would be nice there was an option to choose an alternate actor where your components are when crafting. This way you wont have to drag them to your player to craft.

## Benefit(s)
Less dragging and dropping for players.

## Potential Implementation(s)
Maybe add to the player app?

**Comment by misterpotts** [comment]
Yeah, the Actor crafting app is going to be the place for this. That's the eventual place where recipes will be able to be learned, searched and crafting triggered by users. Adding a place to drop additional component sources makes a lot of sense 👍🏻

**Comment by misterpotts** [comment]
This was partially implemented in `0.9.0` as the API supports the operation, but the UI doesn't yet allow you to set a different source actor. I'll keep this open until that's done.
